Automattic/newspack-newsletters
# Remove Newspack UI from Ads Admin page

### All Submissions:






### Changes proposed in this Pull Request:



Newspack UI will be used in the Newspack Plugin

### How to test the changes in this Pull Request:

1. Visit Newsletters -> Ads page 
2. Observe a regular WP post list UI
3. Create an Ad, verify add creation/editiing/removing works as before

### Other information:
